dotcom-rendering/src/components/SlideshowCarousel.tsx

```diff
@@ -162,38 +195,53 @@ export const SlideshowCarousel = ({
 					);
 				})}
 			</ul>
-			<div css={buttonStyles}>
-				<Button
-					hideLabel={true}
-					iconSide="left"
-					icon={<SvgChevronLeftSingle />}
-					onClick={() => scrollTo('left')}
-					priority="tertiary"
-					theme={
-						previousButtonEnabled
-							? themeButton
-							: themeButtonDisabled
-					}
-					size="small"
-					disabled={!previousButtonEnabled}
-					aria-label="Move image carousel backwards"
-					// TODO: data-link-name="slideshow carousel left chevron"
-				/>
-
-				<Button
-					hideLabel={true}
-					iconSide="left"
-					icon={<SvgChevronRightSingle />}
-					onClick={() => scrollTo('right')}
-					priority="tertiary"
-					theme={
-						nextButtonEnabled ? themeButton : themeButtonDisabled
-					}
-					size="small"
-					disabled={!nextButtonEnabled}
-					aria-label="Move image carousel forwards"
-					// TODO: data-link-name="slideshow carousel right chevron"
-				/>
+			<div css={navigationStyles}>
+				<div css={paginationStyles}>
+					{takeFirst(images, 10).map((image, index) => (
+						<span
+							css={[
+								dotStyles,
+								currentPage === index && activeDotStyles,
+							]}
+							key={image.imageSrc}
+						/>
+					))}
+				</div>
+				<div css={buttonStyles}>
+					<Button
+						hideLabel={true}
+						iconSide="left"
+						icon={<SvgChevronLeftSingle />}
+						onClick={() => scrollTo('left')}
+						priority="tertiary"
+						theme={
+							previousButtonEnabled
+								? themeButton
+								: themeButtonDisabled
+						}
+						size="small"
+						disabled={!previousButtonEnabled}
+						aria-label="Move image carousel backwards"
+						// TODO: data-link-name="slideshow carousel left chevron"
+					/>
+
+					<Button
+						hideLabel={true}
+						iconSide="left"
+						icon={<SvgChevronRightSingle />}
+						onClick={() => scrollTo('right')}
+						priority="tertiary"
+						theme={
+							nextButtonEnabled
+								? themeButton
+								: themeButtonDisabled
+						}
+						size="small"
+						disabled={!nextButtonEnabled}
+						aria-label="Move image carousel forwards"
```

Comment:
> Maybe this and the previous ARIA label could be "view next image in slideshow" or something similar? Thinking this might be more human readable

I used the labels from the existing carousels as inspiration, but I agree it's a little clunky and not very human readable. I've updated with your suggestion, but I think we'll want to revisit as part of our accessibility review. I need to do some reading about accessibility best practices for carousels as I think there's more we can do.